workbench-core/accounts/src/utilities/hostingAccountLifecycleService.ts

```diff
@@ -604,17 +604,17 @@ export default class HostingAccountLifecycleService {
     const {
       [process.env.STATUS_HANDLER_ARN_OUTPUT_KEY!]: statusHandlerArn,
       [process.env.S3_ARTIFACT_BUCKET_ARN_OUTPUT_KEY!]: artifactBucketArn,
-      [process.env.MAIN_ACCT_ENCRYPTION_KEY_ARN_OUTPUT_KEY!]: mainAcctEncryptionArn
+      [process.env.S3_DATASETS_ENCRYPTION_KEY_ARN_OUTPUT_KEY!]: s3DatasetsEncryptionArn
```

Comment:
I think this might actually need to be S3_ARTIFACT_ENCRYPTION_KEY_ARN_OUTPUT_KEY. @nguyen102 or @SanketD92 able to confirm?

Reply:
we need it for provision environment. The environment-files parameter that gets passed during env provisioning is in the artifacts bucket